tests/retrieval/helpers.py

```diff
@@ -1,129 +1,497 @@
-from typing import Callable, List
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from functools import partial
+from typing import Callable, Tuple, Union
 
 import numpy as np
 import pytest
 import torch
+from numpy import array
 from torch import Tensor
 
 from tests.helpers import seed_all
-from torchmetrics import Metric
+from tests.helpers.testers import Metric, MetricTester
+from tests.retrieval.inputs import (
+    _input_retrieval_scores,
+    _input_retrieval_scores_all_target,
+    _input_retrieval_scores_empty,
+    _input_retrieval_scores_extra,
+    _input_retrieval_scores_mismatching_sizes,
+    _input_retrieval_scores_mismatching_sizes_func,
+    _input_retrieval_scores_no_target,
+    _input_retrieval_scores_wrong_targets,
+)
+from torchmetrics.utilities.data import get_group_indexes
 
-seed_all(1337)
+seed_all(42)
```

Comment:
I think this shall be inside a function, this is called only when the module is loaded not function, right?

Reply:
Yes....I just put it because I saw it in other tests. My tests should work with every seed. If you want I can put it in every test function.